order.go

```diff
@@ -0,0 +1,108 @@
+package apiutil
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var (
+	// OrderASC defines the ascending order keyword
+	OrderASC = "asc"
+	// OrderDESC defines the descending order keyword
+	OrderDESC = "desc"
+)
+
+// OrderBy represents ordering information
+type OrderBy struct {
+	OrderItems []Order
+}
+
+// Order represents ordering information for a single key
+type Order struct {
+	Key       string
+	Direction string
+}
+
+// NewOrderBy constructs a basic order by struct
+func NewOrderBy() OrderBy {
+	return OrderBy{
+		OrderItems: []Order{},
+	}
+}
+
+// NewOrder constructs a basic order struct
+func NewOrder(key string, orderDirection string) Order {
+	if orderDirection != OrderASC && orderDirection != OrderDESC {
+		orderDirection = OrderDESC
+	}
+	return Order{
+		Key:       key,
+		Direction: orderDirection,
+	}
+}
+
+// AddOrdering allows to add items to the current ordering
+func (o *OrderBy) AddOrdering(order ...Order) {
+	o.OrderItems = append(o.OrderItems, order...)
+}
+
+// String implements the stringer interface for OrderBy
+func (o OrderBy) String() string {
+	var b bytes.Buffer
```

Comment:
use a [`strings.Builder`](https://golang.org/pkg/strings/#Builder) instead. It's more efficient, but the real benefit is dropping the need for the  `bytes` package from this file